src/documents/tutorial-gettingStartedWithInfusion/BasicComponentCreation-Components.md

```diff
@@ -161,13 +161,13 @@ tutorials.currencyConverterAuto.convert = function (exchangeRate, amount) {
 };
 ```
 
-You'll notice that in this case we have been able to avoid binding to the entire component instance in our public function, and so our standalone public function `tutorials.currencyConverterAuto.convert` 
-is of more general utility than just for building a component method. 
+You'll notice that in this case we have been able to avoid binding to the entire component instance in our public function, and so our standalone public function `tutorials.currencyConverterAuto.convert`
+is of more general utility than just for building a component method.
```

Comment:
Could you fix the bug in the code example code that this refers to. The function takes in `exchangeRate` but then uses `that.options.exchangeRate` internally.